### PR TITLE
fix: add classifiers in setup.py

### PR DESCRIPTION
**Pull request by Zheaoli**
Signed-off-by: Manjusaka <me@manjusaka.me>

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -54,6 +54,19 @@ def get_version():
     ver = version.rsplit(" ", 1)[-1][1:]
     return ver
 
+classifiers = [
+    'Development Status :: Aplha 9',
```
Comment (kemingy) [inline setup.py:58]: Do we need to change this frequently since we may release lots of test versions recently?

**Review by gaocegege — COMMENTED** [review]
Thanks for your contribution! :tada: :+1:

/assign @kemingy @VoVAllen

File: setup.py (hunk 1 above)
Comment (kemingy) [reply setup.py:58]: LGTM

Comment (Zheaoli) [reply setup.py]: Done, PTAL

File: setup.py [hunk 2]
```diff
@@ -54,6 +54,19 @@ def get_version():
     ver = version.rsplit(" ", 1)[-1][1:]
     return ver
 
+classifiers = [
+    'Development Status :: Aplha',
+    'Topic :: Software Development :: Build Tools',
+    'Intended Audience :: Science/Research',
+    'Intended Audience :: Developers',
+    'License :: OSI Approved :: Apache 2.0 License',
```
Comment (kemingy) [inline setup.py:62]: according to https://pypi.org/classifiers/
```suggestion
    'License :: OSI Approved :: Apache Software License',
```

**Review by kemingy — COMMENTED** [review]
The rest part looks good to me.

File: setup.py (hunk 2 above)
Comment (Zheaoli) [reply setup.py]: Done

**Review by kemingy — APPROVED** [review]
LGTM

**Comment by gaocegege** [comment]
Thanks for your contribution! :tada: :+1:

**Pull request merged**